helpers/handlebars/helpers/index.js

```diff
@@ -286,6 +286,15 @@ const helpers = app => ({
 		}
 		return i18n.getInstance(opts.data.local.currentUser)(key, data);
 	},
+	dict: (...keyValues) => {
+		const dict = {};
+		keyValues.forEach((keyValue, index) => {
+			if (!(index % 2)) {
+				dict[keyValue] = keyValues[index + 1];
+			}
+		});
+		return dict;
+	},
```

Comment:
Please document the usage with an examle [in confluence/docs](https://docs.schul-cloud.org/pages/viewpage.action?pageId=113643606#Localization%F0%9F%87%AC%F0%9F%87%A7%F0%9F%87%B5%F0%9F%87%B9%F0%9F%87%B9%F0%9F%87%B7-Handlebars)